content/docs/start/data-pipelines.md

```diff
@@ -1,4 +1,10 @@
-# Data Pipelines
+---
+title: 'Get Started: Data Pipelines'
+description: 'Get started with pipelines in DVC. Learn how to capture, organize,
+version, and reproduce your data science and machine learning workflows.'
+---
+
+# Get Started: Data Pipelines
```

Comment:
> Added a custom title to prefix title with "Get Started:" in search result. Edited page H1 to match. Suggest using this pattern for all Get Started items and Tutorial. Adding prefixes like this should improve clickthrough and reduce bounce by clearly showing the result is a "Get Started"

Oh, I didn't realize we already did that for https://dvc.org/doc/start/data-versioning in fact!

Hmmmm... Is the reason we're changing the H1 that the HTML title is taken from there @jeremydouglass ? If so we could change the engine to use the SEO title instead, and keep a lean H1 in the page body. Please lmk before I request that feature unnecessarily.

Reply:
Actually I checked and the SEO title is what's used for the HTML head title tag. So do we need the H1 title to be the same? I personally prefer the old way but it's not a strong opinion so if there's a reason, even if it's just for a potential benefit, we can def. consider it.

Reply:
The Get Started docs make a better case for using the "Get Started: Data Pipelines" format for both H1 and SEO title. I think you'd agree that's better than "Data Pipelines Get Started". 

For the tutorial, (as in #1856) "Tutorial: Data and Model Versioning" fits the standard for how Google is recommending titles for search results. Another tactic/example, used elsewhere, is the title itself as the prefix, something like "Widgets: Do Cool Widgety Things | Widget Corp." 

Tying the two together, I think it's best for the SEO title (what's clicked on) to match the H1 so there isn't a disconnect on landing (page title matches the search result). So I'm advocating for "Get Started: Data Pipelines" format for both SEO title and H1.

What do you think @jorgeorpinel ?

Reply:
OK sure, if there's a SEO-related reason then it's in scope. Let's do it!